modules/cloud_logs/variables.tf

```diff
@@ -122,3 +122,29 @@ variable "existing_en_instances" {
   default     = []
   description = "List of Event Notifications instance details for routing critical events that occur in your IBM Cloud Logs"
 }
+
+##############################################################################
+# Logs Routing
+##############################################################################
+
+variable "enable_cloud_logs_as_target" {
+  description = "Whether to set IBM Cloud Logs be designated as a target for logs routing in a specific region."
+  type        = bool
+  default     = false
```

Comment:
Is there a reason why we default to false here instead of true?

Reply:
Not sure if there was any specific reason. Changed it to true.